modules/cluster-logging-loki-grafana.adoc

```diff
@@ -0,0 +1,89 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: PROCEDURE
+[id="logging-loki-grafana_{context}"]
+
+= Configuring Grafana
+Grafana is an established web-based dashboard and visualization tool for interacting with Loki.
```

Comment:
We need a strong statement this is not a supported offering and customer assumes full responsibility for its deployment and configuration

Reply:
Is it better to exclude it? @rkratky thoughts?

Reply:
It's potentially valuable info, so I'd suggest adding a link to Additional resources.

Reply:
Placed in additional resources.